ui/src/components/Code/index.jsx

```diff
@@ -40,12 +40,10 @@ export default class Code extends Component {
       ignoreIllegals: true,
     }).value;
 
-    /* eslint-disable react/no-danger */
     return (
       <pre className={classNames(`language-${language}`, className)} {...props}>
-        {code && <code dangerouslySetInnerHTML={{ __html: code }} />}
+        {code && <code>{code}</code>}
```

Comment:
I believe this will render literal HTML, which is not what we want.  I believe the output of `highlight` is escaped and considered safe.

Reply:
Oh sorry, you don't have access to the bug, but this was a quick test to see if this would cause double-escaping (for the reason you cited)! I made it a draft PR because I didn't expect it to be needed, but if it rendered correctly, it would have implied that escaping wasn't being done underneath, and would have been straightforward to mark it as ready for review.